### DISCUSSION TITLE
How to toggle between normal and silent mode via keyboard or command line option?

### DISCUSSION THREAD

**Discussion by Christian79AD** [rcaelers/workrave#451] [Q&A]
Hello.

I love Workrave! :-)

Does a commandline option exist, that allows me to toggle between normal and silent mode? I need to do this often.

If yes, I would create a hotkey (maybe in AutoHotKey or directly in Windows 10) for toggling these two states, which would be faster than using the mouse (and easier for people with repetitive strain injury and/or using speech recognition software like Talon Voice).

I would also appreciate it if you could define this hotkey in Workrave itself.

Best regards,

Christian

**Comment by hikatamika** [comment]
Also curious about this.

As it stands I have an autohotkey script find the applet, right click inside of it, then press the menu item keys in sequence to change modes, but if there's an easier way, that'd be preferred.
